src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/digitaltwinrepositoryprovisioningservice/__init__.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Why are you vendoring an sdk within the CLI? We do not vendor sdks in the CLI. This is permitted in extensions but not in the core CLI.

cc: @tjprescott 

It seems like there is an exception [here](https://github.com/Azure/azure-cli/tree/42046bb50082e1ae14fd74af92325e9b84ef5b09/src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/mgmt_iot_hub_device), which shouldn't be the case.